File: mobile/src/main/java/uk/co/appsbystudio/geoshare/utils/services/StopTrackingService.java
package uk.co.appsbystudio.geoshare.utils.services;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class StopTrackingService extends BroadcastReceiver {

    @Override
    public void onReceive(Context context, Intent intent) {
        Intent service = new Intent(context, TrackingService.class);
        context.stopService(service);
    }
}

File: mobile/src/main/java/uk/co/appsbystudio/geoshare/utils/services/StopTrackingService.kt
package uk.co.appsbystudio.geoshare.utils.services

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class StopTrackingService : BroadcastReceiver() {

    override fun onReceive(context: Context?, intent: Intent?) {
        val service = Intent(context, TrackingService::class.java)
        context?.stopService(service)
    }
}
